src/ripple/basics/random.h

```diff
@@ -67,24 +68,41 @@ inline
 beast::xor_shift_engine&
 default_prng ()
 {
+    // This is used to seed the thread-specific PRNGs on demand
+    static beast::xor_shift_engine seeder = []
+    {
+        std::random_device rng;
+
+        std::uint64_t seed = 0;
+
+        for (int i = 0; i < 16; ++i)
+            seed = (seed << 4) ^ rng();
+
+        // Exceedingly unlikely to happen, but just in case
+        if (seed == 0)
```

Comment:
No change needed, but this could be a `while`. But honestly, this is never going to be hit unless `random_device` is broken.